docs/source/examples/zne_logical_rb_cirq_stim.md

```diff
@@ -0,0 +1,302 @@
+
+```{code-cell} ipython3
+scale_factors = [1, 3, 5, 7]
+fac = mitiq.zne.PolyFactory(scale_factors=scale_factors, order=3)
+scaling = mitiq.zne.scaling.folding.fold_global
+```
+
+```{code-cell} ipython3
+scaled_stim_circuits = []
+for c in filled_circuits:
+    scaled_stim_circuit = []
+    for s in scale_factors:
+        scaled_circuit = mitiq.zne.scaling.fold_global(c, s)
+        scaled_stim_circuit.append(
+            stimcirq.cirq_circuit_to_stim_circuit(scaled_circuit)
+        )
+    scaled_stim_circuits.append(scaled_stim_circuit)
+```
+
+```{code-cell} ipython3
+mitigated_results = np.zeros((trials, len(d_array)))
+
+for di in range(len(d_array)):
+    for t in range(trials):
+        fac.reset()
+        for s, f in zip(scale_factors, scaled_stim_circuits[t]):
+            fac.push(
+                {"scale_factor": s},
+                stim_executor(
+                    f,
+                    p_err,
+                    p_th,
+                    d_array[di],
+                    scale_shots(
+                        device_size, d_array[di], base_shots, n_qubits
+                    ),
+                ),
+            )
+        mitigated_results[t, di] = fac.reduce()
+```
+
+## Plot results
+
++++
+
+Finally, plot the mean unmitigated and ZNE-mitigated expectation values, averaged over the trial circuits, at every code distance simulated. The error bars indicate the standard deviation.
+
+```{code-cell} ipython3
+:tags: [remove-output]
+
+mpl.pyplot.errorbar(
+    d_array,
+    np.mean(mitigated_results, axis=0),
+    yerr=np.std(mitigated_results, axis=0),
+    label="Folding, m = 100",
+)
+
+mpl.pyplot.errorbar(
+    d_array,
+    np.mean(noisy_results, axis=0),
+    yerr=np.std(noisy_results, axis=0),
+    ls="--",
+    marker="o",
+    label="Unmitigated, m = 100",
+)
+
+mpl.pyplot.plot(
+    d_array,
+    np.ones((len(d_array), 1)),
+    ls=":",
+    marker="*",
+    label="Ideal",
+    lw=2,
+)
+
+mpl.rc("font", family="Times New Roman")
+mpl.pyplot.title(
+    r"""Expectation value of $A=|00\rangle\langle00|$ from RB circuits acting on logical qubits""",
+    fontsize=12,
+)
+mpl.pyplot.xticks(ticks = [11, 13, 15, 17, 19, 21])
+
+mpl.pyplot.xlabel(r"Maximum code distance, $d_{i,0}$")
+mpl.pyplot.ylabel(r"Expectation value, $E$")
+
+mpl.pyplot.legend()
+mpl.pyplot.show()
+```
+
+```{figure} ../_thumbnails/zne_stim.png
+---
+
+name: zne-stim-plot
+---
+Plot of the unmitigated and ZNE-mitigated expectation values obtained from executing the logical RB circuits.
+```
+
+We can see from the above plot that the ZNE-mitigated expectation values are closer to the ideal value of 1.0 at every code distance simulated. 
+The effect is more pronounced at lower code distances, which correspond to a higher logical error rate, whereas by $d = 21$ both the mitigated and unmitigated expectation values approach 1.0. 
+We can also think of the results in terms of an effective code distance, where e.g. . 
```

Comment:
uh oh something got dropped here!

Reply:
Changed my mind and removed the half-sentence about effective code distance.